db/migrations/20211019191814-create-answer.js

```diff
@@ -22,10 +22,18 @@ module.exports = {
       questionId: {
         allowNull: false,
         type: Sequelize.INTEGER,
+        references: {
+          model: "Question",
+          key: "id",
+        },
       },
       userId: {
         allowNull: false,
         type: Sequelize.INTEGER,
+        references: {
+          model: "User",
+          key: "id",
```

Comment:
I don't think you need to add the key: "id" because it should default to the PK on the user table